drop-boxes/register-omero-metadata/register-omero.py

```diff
@@ -61,7 +63,10 @@
 INCOMING_DATE_FORMAT = '%d.%m.%Y'
 OPENBIS_DATE_FORMAT = '%Y-%m-%d'
 
-PROPPERTY_FILTER_LIST = ["IMAGE_FOLDER_PATH", "INSTRUMENT_USER", "IMAGING_DATE", "SAMPLE_ID", "OMERO_TAGS", "ETL_TAG"]
+# For fast validation without parser object
+REQUIRED_PROPPERTY_LIST = ["IMAGE_DATA_PATH", "IMAGING_MODALITY", "IMAGED_TISSUE", "INSTRUMENT_MANUFACTURER", "INSTRUMENT_USER", "IMAGING_DATE"]
+# To filter property list before pushing key-value pair to OMERO server
+PROPPERTY_FILTER_LIST = ["IMAGE_DATA_PATH", "INSTRUMENT_USER", "IMAGING_DATE", "SAMPLE_ID", "OMERO_TAGS", "ETL_TAG"]
```

Comment:
It is not 100% clear to me from this description what is filtered out. E.g. is Image_Data_Path necessary but it doesn't end up in omero?

Reply:
NIT: property with a single P

Reply:
correct, it is to filter out the properties that we don't want to put as key-value pairs in OMERO